src/support/utf8.c

```diff
@@ -232,7 +232,14 @@ size_t u8_offset(const char *s, size_t charnum)
     return i;
 }
 
-/* byte offset => charnum */
+/**
+ * @brief      Calculates number of characters in a UTF-8 string
+ * 
+ * @param[in]  iStr UTF-8 encoded string
```

Comment:
Yes - I had moved u8_charnum (which I'd already rewritten and which has already been moved into 0.4 and 0.3.8) to my own utf.h / utf.c files, with my conversion speedups written in C also,
but then people wanted it in pure Julia, so I separated out that part, and moved the new u8_charnum and u16_charnum back into utf8.c... forgot to update the doxygen comment for that function, thanks for catching that!